Assets/UdonSharp/Editor/UdonSharpExpressionCapture.cs

```diff
@@ -2162,7 +2176,7 @@ public void HandleArrayIndexerAccess(SymbolDefinition.COWValue indexerValue, Sym
             _accessValue = ExecuteGetCOW(); // implicitly adds refcount
             cowValue = null; // Move ownership to accessValue here
 
```

Comment:
Should setting accessSymbol also reset accessValue?

Reply:
Yeah, because when access symbol is set it means that the expression has moved to another archetype so it's invalid to keep the value populated. The arrayBacktraceValue is kept around for the cases where it's still valid to keep the value.

Reply:
Sorry, wrong way around - should setting accessValue also reset accessSymbol? Currently, in this PR, it doesn't.

Reply:
Ah yeah that'd make sense to do.

Reply:
Added to the PR.